src/ra_server_proc.erl

```diff
@@ -1188,6 +1206,77 @@ handle_effect(RaftState, {aux, Cmd}, EventType, State0, Actions0) ->
         handle_effects(RaftState, Effects, EventType,
                        State0#state{server_state = ServerState}),
     {State, Actions0 ++ Actions};
+handle_effect(leader, {add_member, Conf, ServerId, Members}, _EventType,
```

Comment:
Should we allow addition of multiple members at once? Should we allow adding and removing members as part of the same effect?